homeassistant/components/hue/v2/helpers.py

```diff
@@ -0,0 +1,21 @@
+"""Helper functions for Philips Hue v2."""
+
+
+def brightness_helper(brightness):
+    """Returns calculated brightness values"""
+
+    if brightness is not None:
+        # Hue uses a range of [0, 100] to control brightness.
+        brightness = float((brightness / 255) * 100)
+
+    return brightness
+
+
+def transition_helper(transition):
```

Comment:
See my previous comment, I suggest it change it to "get_hue_transition"

Reply:
Just an idea for the naming, may be we change it to "normalize_hue_transition" ?